src/styles/contacts.styl

```diff
@@ -0,0 +1,35 @@
+@require 'settings/palette.styl'
+
+.contact
+    display flex
+    height 48px
+    box-shadow 0 1px 0 0 silver
+    padding-left 2rem
+
+    & div
```

Comment:
It's strange to see a rule for an unnamed `div` here when everything else has proper names. The `line-height` rule could be defined on the parent I think, and for the margin I think it would be better with a named class, like `contact-wrapper` or something like that.